otel/starter/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/interceptor"
+
+	otelworkflow "github.com/temporalio/samples-go/otel"
+)
+
+func main() {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	otelworkflow.Setup(ctx)
+	defer otelworkflow.Shutdown(ctx)
+
+	options := client.Options{
+		HostPort:     "localhost:7233",
+		Interceptors: []interceptor.ClientInterceptor{otelworkflow.GetInterceptor()},
+	}
+
+	// The client is a heavyweight object that should be created once per process.
+	c, err := client.Dial(options)
+	if err != nil {
+		log.Fatalln("Unable to create client", err)
+	}
+	defer c.Close()
+
+	workflowOptions := client.StartWorkflowOptions{
+		ID:        "otel_workflowID",
+		TaskQueue: "otel",
+	}
+
+	we, err := c.ExecuteWorkflow(context.Background(), workflowOptions, otelworkflow.Workflow, "Temporal")
+	if err != nil {
+		log.Fatalln("Unable to execute workflow", err)
+	}
+
+	log.Println("Started workflow", "WorkflowID", we.GetID(), "RunID", we.GetRunID())
```

Comment:
The standard Go logger doesn't usually expect key-value pair type logs like structured logging does

Reply:
This line is common to many other examples in the repo like: https://github.com/search?q=repo%3Atemporalio%2Fsamples-go+log.Println%28%22Started+workflow%22&type=code

Reply:
:+1: We get it wrong in those too, oh well. Can ignore.